apps/web/src/hooks/useAuthController.ts

```diff
@@ -1,3 +1,3 @@
-import { useAuthController, applyToken, getTokenPayload, getToken } from '@novu/shared-web';
+import { useAuthController, applyToken } from '@novu/shared-web';
```

Comment:
```suggestion
export { useAuthController, applyToken } from '@novu/shared-web';
```

Reply:
We have a bunch of these `import`/re-`export` patterns for `@novu/shared-web`, I'll suggest we handle them separately in batch.